apps/src/gamelab/GameLabP5.js

```diff
@@ -581,6 +582,19 @@ GameLabP5.prototype.afterSetupComplete = function() {
   this.p5._setupEpiloguePhase2();
 };
 
+GameLabP5.prototype.preloadBackgrounds = function() {
+  this.p5._predefinedBackgrounds = {};
+
+  return new Promise(resolve => {
+    backgrounds.forEach(background => {
+      this.p5.loadImage(background.sourceUrl, image => {
+        this.p5._predefinedBackgrounds[background.legacyParam] = image;
```

Comment:
Will this still work if there is no legacyParam?

Reply:
Also, worth adding a test?

Reply:
If we add more background images, then we will need additional logic to key by name in those cases. I think it's fine to hold off on adding that until we actually add new images, though.

Reply:
👍